architecture/core_concepts/builds_and_image_streams.adoc

```diff
@@ -112,29 +120,35 @@ https://registry.hub.docker.com/u/openshift/origin-custom-docker-builder[Docker
 Hub] as an example implementation of a Custom builder image.
 
 [[image-streams]]
-
 == Image Streams
 
-An _image stream_ can be used to automatically perform an action, such as
-updating a deployment, when a new image, such as a new version of the base image
-that is used in that deployment, is created.
-
-An image stream comprises one or more
+An _image stream_ comprises one or more
 link:containers_and_images.html#docker-images[Docker images] identified by tags.
 It presents a single virtual view of related images, similar to a Docker image
 repository, and may contain images from any of the following:
 
-. Its own image repository in OpenShift's integrated Docker Registry
+. Its own image repository in {product-title}'s
+link:../../architecture/infrastructure_components/image_registry.html#integrated-openshift-registry[integrated
+registry]
 . Other image streams
 . Docker image repositories from external registries
 
-OpenShift components such as builds and deployments can watch an image stream
-to receive notifications when new images are added and react by performing
-a build or a deployment.
+Image streams can be used to automatically perform an action when new images are
+created. Builds and deployments can watch an image stream to receive
+notifications when new images are added and react by performing a build or
+deployment, respectively.
 
-.Image Stream Object Definition
+For example, if a deployment is using a certain base image and a new version of
```

Comment:
s/certain base image/certain image/

Base suggests you're building on top of it, which is rather for builds. Deployments use specific images for running.